src/pages/registration/FileList.tsx

```diff
@@ -0,0 +1,253 @@
+import {
+  Box,
+  Link,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+} from '@mui/material';
+import { useFormikContext } from 'formik';
+import { Trans, useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import { RootState } from '../../redux/store';
+import { AssociatedFile, Uppy } from '../../types/associatedArtifact.types';
+import { licenses, LicenseUri } from '../../types/license.types';
+import { Registration } from '../../types/registration.types';
+import { dataTestId } from '../../utils/dataTestIds';
+import {
+  associatedArtifactIsFile,
+  isDegreeWithProtectedFiles,
+  isEmbargoed,
+  isTypeWithFileVersionField,
+  userCanUnpublishRegistration,
+} from '../../utils/registration-helpers';
+import { administrativeAgreementId, FilesTableRow } from './files_and_license_tab/FilesTableRow';
+import { HelperTextModal } from './HelperTextModal';
+
+interface FileListProps {
+  title: string;
+  files: AssociatedFile[];
+  uppy: Uppy;
+  remove: (index: number) => any;
+  name: string;
```

Comment:
Litt mer eksplisitt navn? Tenkte først dette var headingen til fil-lista eller noe sånt, men ser det brukes for navn til formik-verdi
```suggestion
  baseFieldName: string;
```